libraries/standard/mqtt/utest/mqtt_lightweight_utest.c

```diff
@@ -424,7 +424,7 @@ void test_MQTT_GetConnectPacketSize( void )
     status = MQTT_GetConnectPacketSize( &connectInfo, NULL, &remainingLength, &packetSize );
     TEST_ASSERT_EQUAL_INT( MQTTBadParameter, status );
 
-    /* Connect packet too large. */
+    /* Test a will message payload length beyond 16 bits. */
```

Comment:
This comment isn't entirely correct, since `UINT16_MAX` fits within 16 bits.  It would be more correct to say "Test a will message payload that is too large".

Reply:
The payload length is willInfo.payloadLength = UINT16_MAX + 2; in this test